lib/core/presentation/widgets/bottom_bar/bottom_bar_widget.dart

```diff
@@ -14,8 +14,10 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:flutter_screenutil/flutter_screenutil.dart';
 import 'package:flutter_vibrate/flutter_vibrate.dart';
 
-class BottomBar extends StatefulWidget {
+class BottomBar extends StatefulWidget { 
   const BottomBar({super.key});
+  
+  static double get bottomBarHeight => Platform.isIOS ? 90.h : 70.h;
```

Comment:
Why we need this ?

Reply:
the current bottom bar hide a part of scrollview in a screen so have to get this value to add extra space under the scrollview, so that user can scroll and see fully section